core/src/display_object/edit_text.rs

```diff
@@ -315,6 +315,7 @@ impl<'gc> EditText<'gc> {
         drop(edit_text);
 
         self.relayout(context);
+        self.on_changed(context);
```

Comment:
Looks like `onChanged` should only fire from explicit user input in the text field, and not from programmatic changes (i.e. `txt.text = "FOO";` does not fire the onChanged handler).